monty/utils/helpers.py

```diff
@@ -106,3 +108,13 @@ async def internal_task() -> None:
 
     start = loop.time()
     return scheduling.create_task(internal_task())
+
+
+def utcnow() -> datetime.datetime:
+    """Return the current time as an aware datetime in UTC."""
+    return datetime.datetime.now(datetime.timezone.utc)
+
+
+def fromisoformat(timestamp: str) -> datetime.datetime:
+    """Parse the given ISO-8601 timestamp to a datetime object."""
+    return dateutil.parser.isoparse(timestamp)
```

Comment:
ironically, it looks like this may return naive datetimes.

Reply:
If the iso timestamp doesn't contain a timezone, yeah. I haven't seen any places that receive timestamps without a tz, other than the pypi module, where this is already handled.

I suppose we could just assume UTC for native datetimes here, so that this always returns an aware datetime?

Reply:
oki
https://github.com/onerandomusername/monty-python/pull/240/commits/46f922670d1ffb7ce9d065ab0fa6d450deae196f